README.md

```diff
@@ -1 +1,63 @@
 # java-server
+
+A program that can run either an echo server or a http server.
+
+## Dependencies
+
+This is a Java v14 project using Maven v4.0.0
+
+## Setup
+
+### Clone down the project
+
+```shell
+git clone https://github.com/karl-thomas/java-server.git
+```
+
+### Build the project
+
+```shell
+mvn package
+```
+
+## Running the Project
+
+In order to run the project successfully, you need to supply command line arguments when running the Main class.
+
+The first argument is for the port, and the second is for denoting between running an `echo` server, or a `http` server.
+
+### Echo
+
+To run the echo server you could run
+
+```shell
+java -jar target/java-server-1.0-SNAPSHOT.jar 5000 echo
+```
+
+This will start an echo server on port 5000, and it will echo back anything that you sent to it.
+
+### HTTP
+
+To run the HTTP server you could run
+
+```shell
+java -jar target/java-server-1.0-SNAPSHOT.jar 5000 http
```

Comment:
Nice. I like that you give the specific command that you need to use to run the app or the tests, for example. 

A bonus would be to provide an example of the specific syntax you could type to make the Get call to echo back what you type in.